pkg/util.go

```diff
@@ -77,6 +77,30 @@ func IsTime(stringDate string) bool {
 	return false
 }
 
+func CreateGitIgnore(logger *zap.Logger, filePath string) error {
+	gitIgnoreData := []byte("./reports/\n")
```

Comment:
The path of the reports directory is incorrect

Reply:
tree -a
.
├── .gitignore
├── reports
│   ├── test-run-0
│   │   └── test-set-0-report.yaml
│   └── test-run-1
│       └── test-set-0-report.yaml
└── test-set-0
    ├── mocks.yaml
    └── tests
        ├── test-1.yaml
        ├── test-2.yaml
        └── test-3.yaml

5 directories, 7 files
If this is the folder structure of the keploy folder, could you please explain how is the path to the reports folder incorrect?